File: fix_html_2.py
content = open(r'c:\projects\we_crm\webpage\src\app\dashboard\service-details\service-details.html', encoding='utf-8').read()
content = content.replace('/ autocomplete=\"new-password\">', 'autocomplete=\"new-password\" />')
open(r'c:\projects\we_crm\webpage\src\app\dashboard\service-details\service-details.html', 'w', encoding='utf-8').write(content)
print("Fixed syntax")

File: fix_html_2.js
const fs = require('fs');
const file = 'c:/projects/we_crm/webpage/src/app/client/client-profile.html';
let content = fs.readFileSync(file, 'utf8');
content = content.replace(
  '              </div>\r\n            </div>\r\n            \r\n            <div class=\"documents-list\">',
  '              </div>\r\n              </div>\r\n            </div>\r\n            \r\n            <div class=\"documents-list\">'
);
fs.writeFileSync(file, content);
console.log('Done!');
